Modules/ThirdParty/GoogleTest/src/itkgoogletest/googletest/CMakeLists.txt

```diff
@@ -337,3 +337,7 @@ if (gtest_build_tests)
   py_test(gtest_xml_output_unittest --no_stacktrace_support)
   py_test(googletest-json-output-unittest --no_stacktrace_support)
 endif()
+
+# Begin ITK additions
+mark_as_advanced(GTEST_HAS_ABSL)
+# End ITK additions
```

Comment:
Would it be possible to move those ITK additions to the _ITK specific_ CMakeLists.txt at "Modules/ThirdParty/GoogleTest", https://github.com/InsightSoftwareConsortium/ITK/blob/ce74ac3181cd4f6adb1d7680f793a74908549e99/Modules/ThirdParty/GoogleTest/CMakeLists.txt ?  If so, I think that would be preferable 🤷

Reply:
Yes!